README.md

```diff
@@ -18,8 +18,41 @@ A integração do módulo da Vindi permite criação e gestão de planos e assin
 - Conta ativa na [Vindi](https://www.vindi.com.br "Vindi").
 
 # Instalação
-
-# Atualização
+É possível realizar a instalação do módulo da Vindi para Magento 2 manualmente, via [Git](https://github.com) ou via [Composer](https://getcomposer.org).
+
+#### Via [composer](https://getcomposer.org)
+- Vá até o diretório raíz do Magento e adicione o módulo
+> `composer require vindi/vindi-magento2`
+- Atualize os módulos disponíveis do Magento
+> `bin/magento setup:upgrade`
+- O módulo **Vindi_Payment** deverá ser exibido na lista de módulos do Magento
+> `bin/magento module:status`
+
+#### Via [git](https://github.com)
+- Vá até o diretório raíz do Magento e adicione o módulo
+> `git clone git@github.com:vindi/vindi-magento2.git app/code/Vindi/Payment/`
+- Atualize os módulos disponíveis do Magento
+> `bin/magento setup:upgrade`
+- O módulo **Vindi_Payment** deverá ser exibido na lista de módulos do Magento
+> `bin/magento module:status`
+
+#### Via [.zip](https://github.com/vindi/vindi-magento2/archive/master.zip)
+- Crie a(s) seguinte(s) pasta(s) dentro da pasta **app** no diretório raíz do Magento
+> `code->Vindi->Payment`
+- Faça o download do [.zip](https://github.com/vindi/vindi-magento2/archive/master.zip)
+- O caminho deve ser **app->code->Vindi->Payment**
+- Extraia os arquivos do **.zip** na pasta **Payment**
+- No diretório raíz, atualize os módulos disponíveis do Magento
+> `bin/magento setup:upgrade`
+- O módulo **Vindi_Payment** deverá ser exibido na lista de módulos do Magento
+> `bin/magento module:status`
+
+# Configuração
+1. Configurando sua conta Vindi
+    - Em *Vindi -> Configuração* selecione o modo de operação e informe a chave da API de sua conta Vindi.
```

Comment:
Esta um pouco confuso quais informações tem que ser extraidas na **plataforma Vindi** e no `admin do Magento`.

Reply:
Também fiquei um pouco confusa. Se for um passo a passo, acho que vale a pena descrever onde será realizada cada etapa e colocar em ordem, qual é o primeiro e os seguintes passos. 

Por exemplo: 
- No admin do Magento, vá em Vindi -> Configuração, selecione o modo de operação e informa a chave de API da sua conta Vindi.
- Você deve copiar o link de configuração dos Webhooks para inserir na plataforma Vindi. Acesse a [plataforma](http://app.vindi.com.br), vá em Configurações -> Webhooks, depois clique em "Novo webhook" e cole o link copiado no campo de URL. Por fim, "Criar Webhook".

Essas são só sugestões, não sei qual passo a passo exatamente. Se for o caso, posso auxiliar de forma mais detalhada, ao saber mais os detalhes.